src/modules/sbstudio/plugin/operators/utils.py

```
@@ -192,6 +193,23 @@ def export_show_to_file_using_api(
     else:
         cameras = None
 
+    # get terrain
+    export_terrain = settings.get("export_terrain", False)
+    if export_terrain:
+        log.info("Exporting terrain, please wait...")
+        try:
+            terrain = get_terrain_from_context(context)
+        except RuntimeError as ex:
+            log.error(f"Failed to export terrain: {ex}")
```

Ruff (0.16.1)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)